source/_patterns/01-elements/link/link.scss

```diff
@@ -26,42 +27,40 @@
 			color: initial;
 		}
 	}
+	&[data-variant="internal"] {
+		@include icon(glyph(link), 20, "outline", "after");
+	}
+	&[data-variant="external"] {
+		@include icon(glyph(link-external), 20, "outline", "after");
+	}
 
+	&[data-variant="internal"],
+	&[data-variant="external"] {
+		color: #282d37;
```

Comment:
to me it seems like that the color isn't related to the variant, but the size (#282d37 for medium and #42464f for small).

Reply:
Fixed